src/SquareCustomer.php

```diff
@@ -107,7 +107,7 @@ public function save()
      * @return \Nikolag\Square\Models\Transaction
      * @throws \Nikolag\Square\Exception on non-2xx response
      */
-    public function charge(float $amount, string $card_nonce, string $location_id)
+    public function charge(float $amount, string $currency, string $card_nonce, string $location_id)
```

Comment:
currency should be the last parameter because it should be optional, if it's not supplied, u should go with USD